lib/_http_outgoing.js

```diff
@@ -464,10 +468,6 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
       else
         len = chunk.length;
 
-      if (this.connection && !this.connection.corked) {
-        this.connection.cork();
-        process.nextTick(connectionCorkNT, this.connection);
-      }
```

Comment:
It already happens for some chunked responses, now it's for all chunked writes. It makes it more efficient this way 😄